documentation/operators/src/introduction.md

```diff
@@ -0,0 +1,41 @@
+# Introduction
+
+This is Nym's operators guide, containing information and setup tutorials about the various pieces of Nym mixnet infrastructure nodes, such as mix node, gateway and network requester as well as Nyx chain validators.
+
+If you are new to Nym and want to learn about the mixnet, how to integrate with the network, developer tutorials and quickstart guides, check out the [Developer Portal](https://nymtech.net/developers/).
+
+If you want to go deeper into Nym architecture and various software such as different, application clients, existing applications like the desktop wallet, mixnet explorer or SDKs, visit [Nym's technical documentation](https://nymtech.net/docs/).
```

Comment:
This sentence needs tweaking, doesn't quite make sense.

Reply:
see now